components/ResultTable/matchResultTable.data.js

```diff
@@ -0,0 +1,96 @@
+export const testMatch = {
+  participants: [
+    {
+      id: 5678,
+      name: 'Mark Brouch',
+      rank: 32
+    },
+    {
+      id: 1234,
+      name: 'Brandon Hedrick',
+      rank: 22
+    }
+  ],
+  id: 'm123',
+  bestOf: 3,
+  winner: 1234,
+  sets: [
+    {
+      winner: 1234,
+      bestOf: 7,
+      games: [
+        {
+          winner: 1234,
+          for: 7,
```

Comment:
Do we need a `for` since it's always 7?

Reply:
So I thought about that. I'm trying to make this as generic as possible. In air hockey you could be playing to 9 or 11 (in rare cases). Also If we wanted to use this for other sports or games we would need to support different values in `for`. (eg Tennis 6-4 or 7-5)

Reply:
good point. proceed
![proceed](https://media.giphy.com/media/3o6Ztc1rWLboGL6wiA/giphy.gif)